### PR TITLE
New self hosted adding custom urls section

### PR DESCRIPTION
**Pull request by SuDa2103**

### CODE REVIEW

**Comment by michaelprichardson** [comment]
GitLive is missing an 'i' on the last line
Screenshot 2021-05-17 at 14 57 40

**Comment by SuDa2103** [comment]
> GitLive is missing an 'i' on the last line
> Screenshot 2021-05-17 at 14 57 40
> 
> Also, I think they are just OAuth applications, not OAuth 2.0

Changed both those, thanks!

File: _sections/self-hosted-admin-android-studio.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ Finally, you will see a screen showing you that GitLive has been installed succe
 
 ![Successful Self Hosted Installation](/uploads/gsh-successful.jpg "Successful Self Hosted Installation"){:class="screenshot"}
 
+### Adding a Custom URL
+You can also add a custom URL that is accessible from the internet. If your remote URL is not accessible from the internet via HTTP or HTTPS, this custom URL will replace the remote URL with another one, so that our server can access the API.
```
Comment (FredCoen) [inline _sections/self-hosted-admin-android-studio.md:54]: If your remote URL is not accessible from the internet via HTTP or HTTPS, you can add a custom URL that is accessible from the internet. This URL will be used for communication between your GitLab instance and our servers.

File: _sections/self-hosted-admin-android-studio.md [hunk 2]
```diff
@@ -50,6 +50,13 @@ Finally, you will see a screen showing you that GitLive has been installed succe
...
+
+There will be a prompt in the footer of the GitLive pane that says "Specify self-hosted GitLab instance URL", this will take you to a webpage where you can add a different URL for all of the requests done by the server.
```
Comment (FredCoen) [inline _sections/self-hosted-admin-android-studio.md:56]: To add your custom URL follow the prompt in the footer of the GitLive pane in your IDE which states "Specify self-hosted GitLab instance URL". You will be redirected to a page where you can enter your custom URL.

**Review by FredCoen — CHANGES_REQUESTED** [review]

**Pull request merged**